test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -164,6 +169,7 @@ describe('View existing record,', function() {
             keys.push(testParams.key.name + testParams.key.operator + testParams.key.value);
             browser.ignoreSynchronization=true;
             var url = browser.params.url + "/record/#" + browser.params.catalogId + "/product-record:" + testParams.table_name + "/" + keys.join("&");
+            testParams.citationParams.permalink = url;
```

Comment:
Is there a better way that we could test this? The definition of `permalink` is separated from other attributes and it's going to be difficult to follow the test case. Why not just test it with the current browser location?

Reply:
I didn't even consider that case. That's the right thing to test. I will update the other test case with the same.